### DISCUSSION TITLE
Assessing the impact of a new channel

### DISCUSSION THREAD

**Discussion by ibrahim-taher** [google/lightweight_mmm#229] [General]
Hello!

I have been using lightweight mmm for several months and have had a great time leveraging it. I've recently run into a problem and could use some help thinking through it. I have a new channel that I want to add that was started last September. My dataset goes back to the beginning of 2021. When I include the channel in my model the contribution is quite low. This makes sense as regressions are generally global, so if a variable is generally 0 and didn't cause an insanely high lift, the model is not going to capture that relationship.

I've read that others have used regressions with time varying coefficients to deal with this issue, but there aren't any frameworks that have time varying coefficients AND adstock and saturation built in.

I'm wondering if there is a simple-ish approach to solve for this. One thing I've read about is a rolling regression. It's definitely possible to do a rolling regression with lightweight_mmm as all I would do is expose the model to different time windows.

**Comment by statm3n** [comment]
Thanks for raising this question. Imagine the idea case that your model and new channel are well fitted, would it make more sense to calculated the attribution only using the period when new channel are shown? The point is you could select a period or several periods of time span to calculate channel's contribution as long as it makes sense to your case.

**Reply by ibrahim-taher** [reply]
Yea this is what we ended up doing, but I have concerns with doing this because the channel has only been around for a few months and best practice is to train MMMs on at least 2 years of data.
